mkdocs.yml

```diff
@@ -167,7 +167,9 @@ nav:
       - Permissioning:
         - Create a permissioned network: Tutorials/Permissioning/Create-Permissioned-Network.md
         - Get started with onchain permissioning: Tutorials/Permissioning/Getting-Started-Onchain-Permissioning.md
-        - Upgrade the permissioning contract: Tutorials/Permissioning/Upgrade-Permissioning-Contract.md
```

Comment:
You have to add a redirect at the end of this config file for this page change. Point `Tutorials/Permissioning/Upgrade-Permissioning-Contract.md` to `Tutorials/Permissioning/Upgrading Permissioning Contracts/Upgrade-Permissioning-Contract-v1.md`